repo-index.yml

```diff
@@ -409,19 +409,19 @@ plugins:
   - contact: sdawson@pivotal.io
     name: Samuel Dawson
   binaries:
-  - checksum: 05021d44a70f063b61c99bdc627ee0e1382a829e
+  - checksum: 0fcb8413bd9cfb372d106f32bd657c03cd53c800
     platform: osx
-    url: https://s3.amazonaws.com/cfdev-prod/cfdev/cfdev-v0.0.13-rc.160-darwin
-  - checksum: 0ebe0d505f5fda4913ba22e3212a285df0cbe094
+    url: https:/d3p1cc0zb2wjno.cloudfront.net/cfdev/cfdev-v0.0.14-rc.33-darwin
```

Comment:
It looks like this is failing in CI because it's missing a forward slash (`https:/` instead of `https://`).

Reply:
Complete apologies, please review now.